### DISCUSSION TITLE
change color of crop area

### DISCUSSION THREAD

**Discussion by danarosca** [ValentinH/react-easy-crop#370] [General]
hi guys. I'm trying to change  the color property of reactEasyCrop_CropArea or any other css property
 I use jss so I the following code:

styles.ts:
cropAreaClassName: {
      color: 'transparent',
    },

and in my component:
const clas = {
    cropAreaClassName: classes.cropAreaClassName
  };

and after that in <Cropper classes={clas} ..>

it doesn't work and I've tried with rgba too. What am I doing wrong? Please help :)

**Comment by ValentinH** [comment]
When you inspect with the devtools, do you see the className beeing applied at all? It could be an injection order issue.

If you can create a demo of the issue on codesandbox, I will help you.

**Comment by danarosca** [comment]
I’ve managed to solve it using styles prop instead of classes. The problem
with classes was that only the props which wasn’t in initial css would
apply. But with styles I’ve managed to overwrite the initial css. Have to
say that I am not a master of css so it took me some time.

**Reply by ValentinH** [reply]
Yes, so it was probably due to an injection order issue.
See https://v4.mui.com/styles/advanced/#css-injection-order
